src/client/index.js

```diff
@@ -488,7 +489,7 @@ function runquery() {
 $('button[type=submit]').click(e => {
     e.preventDefault();
     $('#synthesize-bar .query-alert').removeClass('fade').alert('close');
-    $('form').find('input, textarea, button, select').prop('disabled', true);
```

Comment:
What is the reason for this change?

Reply:
Oh... I must've made a mistake. I changed it back with the last commit.